index.html

```diff
@@ -11,18 +11,179 @@
+        <nav id="nav" class="d-flex">
             <a href="#" class="logo">
                 <p>Bonjour</p>
             </a>
             <div><img src="./images/menu.svg" alt="icon_menu" id="menu"></div>
         </nav>
     </header>
     <main>
-        <section class="hero">
-            <div class="hero-content">
+        <!-- hero section -->
+        <section class="hero d-flex flex-column">
+            <div class="header-shapes">
+                <div class="hero-content">
+                    <div class="hero-title ">
+                        <h1>Hello there <br> I'm Zuhaib <br> Glad to see you!</h1>
+                    </div>
+                    <div class="hero-para ">
+                        <p>
+                            I am a Software Developer! I will help you in materializing your dream product, feature or
+                            website.
+                            have a look at my work and experience and if you touched by the work i have done, then
+                            contact
+                            me
+                            by
+                            clicking the icons below.
+                        </p>
+                    </div>
+                    <div>
+                        <p class="hero-social">
+                            let's connect
+                        </p>
+                        <ul class="hero-icons d-flex">
+                            <li>
+                                <a target="_blank" href="https://twitter.com/Zuhaib042" rel="noopener noreferrer"><img
+                                        src="./images/twitter.svg" alt="twitter-logo"></a>
+                            </li>
+                            <li>
+                                <a target="_blank" href="https://www.linkedin.com/in/zuhaib-amjad-488914234/"
+                                    rel="noopener noreferrer"><img src="./images/linkedin.svg" alt="linkedIn-logo"></a>
+                            </li>
+                            <li>
+                                <a href="#" rel="noopener noreferrer"><img src="./images/medium.svg"
+                                        alt="medium-logo"></a>
+                            </li>
+                            <li>
+                                <a target="_blank" rel="noopener noreferrer" href="https://github.com/Zuhaib042"><img
+                                        src="./images/github.svg" alt="github-logo"></a>
+                            </li>
+                            <li>
+                                <a href="#" rel="noopener noreferrer"><img src="./images/good.svg"
+                                        alt="angellist-logo"></a>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <!-- works section -->
+
+        <section class="works">
+            <h1 class="works-title">My Projects</h1>
```

Comment:
- [ ] Please Try to use headings properly to improve your page's accessibility and for SEO reasons. so you need to have one `h1` tag on your pages and other headings can be used multiple times on a page. you got this 👍